Development.md

```diff
@@ -0,0 +1,44 @@
+# Developing lsif-typescript
+
+## References
+
+- VS Code is a good reference point for the "correct" behavior
+  for code navigation functionality.
+- Keep a local checkout of Microsoft/TypeScript for checking out code examples.
+  Most TypeScript APIs are lacking in documentation,
+  so reading the code is the best way to understand how things work.
+
+## Running tests
+
+```sh
+npm run test
+```
+
+```
+npm run update-snapshots
```

Comment:
Does this require having lsif-java installed?

Reply:
No, we have this logic in the test code. https://sourcegraph.com/github.com/sourcegraph/lsif-typescript/-/blob/src/main.test.ts?L101:10

Reply:
Are there any other environment requirements we should document? Specific node version etc etc

Reply:
I don't think we need a specific node version, but we can add that later as a config file (vs just having it in the docs).